src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+
+  &__wrap {
+    display: flex;
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__wrap--4 :nth-child(-n + 4) {
```

Comment:
You don't need additional class such as `__wrap`. It should be the same functionality as in the stars task, if the modifier is `stars--3` you should show 3 active stars